app/views/groups/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="category">
+<h1 class="text-center my-4">Categories</h1>
+<%= render "group" %>
+<%= link_to "Add New Categories", new_group_path, class: "btn mt-3 btn-outline-success"%>
+</div>
```

Comment:
**OPTIONAL** - You have done well so far, but please note that according to the project requirements, you are to style your project according to the given design, doing this would give confidence to potential recruiters in your ability to replicate any design they require you to. Please style this page as it is in the given design.

Your design:
![ergaerg](https://github.com/manzitresor/budget_app/assets/73607512/d047bf57-0027-4a41-9527-169936f5a6d1)